pkg/collector/corechecks/servicediscovery/module/envs.go

```diff
@@ -30,6 +33,36 @@ const (
 	memFdMaxSize = 65536
 )
 
+const (
+	// maxSizeEnvsMap - maximum number of returned environment variables
+	maxSizeEnvsMap = 400
+	// defSizeEnvsMap - default size for environment variables map
+	defSizeEnvsMap = 200
+	// defSizeReadBuf - default buffer size for reading from proc/pid/environ
+	defSizeReadBuf = 2048
+	// defSizeEnvBuf - default buffer size for storing processed environment variable
+	defSizeEnvBuf = 100
+	// envVarDdService - name of target environment variable DD_SERVICE
+	envVarDdService = "DD_SERVICE"
+	// envVarDdTags - name of target environment variable DD_TAGS
+	envVarDdTags = "DD_TAGS"
+	// envVarDdInjectionEnabled - name of target environment variable DD_INJECTION_ENABLED
+	envVarDdInjectionEnabled = "DD_INJECTION_ENABLED"
+	// envVarDiscoveryEnabled - name of target environment variable DD_DISCOVERY_ENABLED
+	envVarDiscoveryEnabled = "DD_DISCOVERY_ENABLED"
+	// envVarOtelServiceName - name of target environment variable OTEL_SERVICE_NAME
+	envVarOtelServiceName = "DD_OTEL_SERVICE_NAME"
```

Comment:
Indirection with constants for these don't seem to add much value. The strings could be used directly instead.

Reply:
Without constants these strings are duplicated in functional code and in unit tests.

Reply:
I refactored the code to add the `targetenvs` package, but still use exported string variables to have a single source of information regarding target environment variables.